docs/devsite-help/support.md

```diff
@@ -0,0 +1,223 @@
+# Support
+
+The vast majority of the code in the Google Cloud Libraries is autogenerated. If
+you're running into problems, it *probably* isn't in that code (although that's
+always a possibility). The problems we see being reported usually fall into a
+few different buckets, described below.
```

Comment:
Maybe link here to the troubleshooting page.

Reply:
Done.